readme-template.md

```diff
@@ -0,0 +1,107 @@
+# Repo Title
+
+<!-- Not all of these are essential but keep this in mind for the repo you are using -->
+![Build badge]() ![Test Coverage]() ![npm package]()
+
+<!--Short sentence on what the repo does-->
+
+<!-- possibly list experts in this repo for who to go to when encountering a problem-->
+
+##Contents
+* [Further Documentation](#documentation)
+* [System Overview](#overview)
+* [Installation](#installation)
+* [Running Locally](#running)
+* [Testing](#testing)
+	* [Automated Testing](#automated)
+	* [Manual testing](#manual)
+* [Script cheat sheet](#cheat-sheet)
+* [Built With](#built-with)
+* [Deploying](#deploy)
+* [Linting](#linting)
+* [Contributing Rules](#rules)
+* [Common Questions](#faq)
+
+<a name="documentation"></a>
+## Further Documentation
+
+* **[System Roadmap]()**
+* **[Regression Testing Checklist]()**
+* **[Change Log]()**
+
+Include any other relevant documentation in this section.
+
+<a name="overview"></a>
+## System Overview
+
+* More in-depth overview of the system, what this is trying to accomplish and how it achieves its goal.
+* Keep it concise, write this section like the first paragraph of a blog post, what is the point of this repo?
+
+<a name="installation"></a>
+## Installation
+
+* Recordit of installation setup for new users
```

Comment:
I'd remove the record it mention here and in the section below, These could quickly become out of date. There is also a chance that we will record something as part of the process that we don't want to be public (certain config vars for example)